lib/measures/urban_geometry_creation/measure.rb

```diff
@@ -52,23 +52,24 @@ def description
   def modeler_description
     return ""
   end
-  
-  # define the arguments that the user will input
+
   def arguments(model)
-    args = OpenStudio::Ruleset::OSArgumentVector.new
-    
+  ##
```

Comment:
This comment should be outside of the function for rdoc right?  We don't normally rdoc measure functions since they all have the same signature